pwa-devdocs/src/technologies/basic-concepts/app-shell/index.md

```diff
@@ -44,9 +44,12 @@ The following is a simple example of an application shell:
 
 
     <script>
-        if (process.env.SERVICE_WORKER && 'serviceWorker' in navigator) {
+        const { navigator } = globalThis;
+        const { serviceWorker } = navigator || {};
+
+        if (process.env.SERVICE_WORKER && serviceWorker) {
             window.addEventListener('load', () => {
-                navigator.serviceWorker
+                serviceWorker
```

Comment:
Should the line above this change too?

Reply:
No, this whole block of code only makes sense in a browser environment anyway. The server won't have a service worker, nor a window to attach listeners to (see L48).